tools/c7n_guardian/c7n_guardian/cli.py

```diff
@@ -288,14 +302,15 @@ def enable_region(master_info, accounts_config, executor, message, region):
                if account['account_id'] not in active_ids]
 
     log.info("Region:%s Accepting %d invitations in members", region, len(members))
-
+   
     with executor(max_workers=WORKER_COUNT) as w:
         futures = {}
         for a in accounts_config['accounts']:
             if a == master_info:
                 continue
             if a['account_id'] in active_ids:
                 continue
+            time.sleep(5)
```

Comment:
the sleep in a loop here is bit problematic, if i have a hundred accounts, i have to wait 500 seconds, at most we should sleep once for all accounts. i haven't actually seen this behavior yet, but i've only found a delete member need on invalid email registration addresses.